apitests/error_handler_test.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from flask_testing import TestCase
+from flask_api import status
+
+
+class ErrorHandlerTest(TestCase):
```

Comment:
Please fix linter error or change linter rule: apitests/error_handler_test.py:9:4: R0201: Method could be a function (no-self-use)